JavaScriptSPA/authPopup.js

```diff
@@ -14,10 +14,23 @@ function signIn() {
         
     }).catch(function (error) {
       console.log(error);
+
+      // Error handling
+      if (error.errorMessage) {
+        // Check for forgot password error
+        // Learn more about AAD error codes at https://docs.microsoft.com/en-us/azure/active-directory/develop/reference-aadsts-error-codes
```

Comment:
@derisen Always strip the locale identifier from Microsoft URLs.

Reply:
sorry, this one slipped away -will remove this in the next pr